### ISSUE TITLE
403 Error while posting comment

### ISSUE THREAD

**Issue by sumeetbits** [typepad/php-typepad-api#3]
Hello,

I am getting 403 Forbidden message while adding comments to  the post...

I am using postToComments() function to post comment..

makeCommentPreview() function is working fine with same parameters passed to postToComments() function..

Is there any setting required in config file? like TP_ACCESS_TYPE?

Please help!

**Comment by kshay** [comment]
Yes, to post comments to a TypePad blog you should be using a TP_ACCESS_TYPE of 'typepad_full'. (After you make this change you'll have to go through the authorization/login steps again.)

**Comment by sumeetbits** [comment]
I have tried this step earlier but its not working.

I changed the TP_ACCESS_TYPE to 'typepad_full' in the config file.

Thanks for reply..

**Comment by sumeetbits** [comment]
Is there any other solution?
